EuclideanGeometry/Foundation/Axiom/Linear/Ray_ex2.lean

```diff
@@ -15,30 +15,30 @@ theorem every_pt_lies_on_seg_of_source_and_target_lies_on_seg {seg₁ seg₂ : S
 
 /- Given two segments $seg_1$ and $seg_2$, if the source and the target of $seg_1$ both lie in the interior of $seg_2$, and if $A$ is a point on $seg_1$, then $A$ lies in the interior of $seg_2$. -/
 theorem
-every_pt_lies_int_seg_of_source_and_target_lies_int_seg : sorry := sorry
+every_pt_lies_int_seg_of_source_and_target_lies_int_seg {seg₁ seg₂ : Seg P}(h: seg₁.source LiesInt seg₂∧seg₁.target LiesInt seg₂){A : P}(ha: A LiesOn seg₁): A LiesInt seg₂:= sorry
 
 /- Given two segments $seg_1$ and $seg_2$, if the source and the target of $seg_1$ both lie on $seg_2$, and if $A$ is a point in the interior of $seg_1$, then $A$ lies in the interior of $seg_2$. -/
 theorem
-every_int_pt_lies_int_seg_of_source_and_target_lies_on_seg : sorry := sorry
+every_int_pt_lies_int_seg_of_source_and_target_lies_on_seg {seg₁ seg₂ : Seg P}(h: seg₁.source LiesOn seg₂ ∧ seg₂.source LiesOn seg₂){A : P}(ha: A LiesInt seg₁): A LiesInt seg₂:= sorry
 
 /- Given a segment and a ray, if the source and the target of the segment both lie on the ray, and if $A$ is a point on the segment, then $A$ lies on the ray. -/
 theorem
-every_pt_lies_on_ray_of_source_and_target_lies_on_ray : sorry := sorry
+every_pt_lies_on_ray_of_source_and_target_lies_on_ray {seg : Seg P}{ray : Ray P}(h: seg.source LiesOn ray ∧ seg.target LiesOn ray){A : P}(ha: A LiesOn seg): A LiesOn ray:= sorry
 
 /- Given a segment and a ray, if the source and the target of the segment both lie in the interior of the ray, and if $A$ is a point on the segment, then $A$ lies in the interior of the ray.-/
 theorem
-every_pt_lies_int_ray_of_source_and_target_lies_int_ray : sorry := sorry
+every_pt_lies_int_ray_of_source_and_target_lies_int_ray {seg : Seg P}{ray : Ray P}(h: seg.source LiesInt ray ∧ seg.target LiesInt ray){A : P}(ha: A LiesOn seg): A LiesInt ray:= sorry
 
 /- Given a segment and a ray, if the source and the target of the segment both lie on the ray, and if $A$ is a point in the interior of the segment, then $A$ lies in the interior of the ray. -/
-theorem every_int_pt_lies_int_ray_of_source_and_target_lies_on_ray : sorry := sorry
+theorem every_int_pt_lies_int_ray_of_source_and_target_lies_on_ray {seg : Seg P}{ray : Ray P}(h: seg.source LiesOn ray ∧ seg.target LiesOn ray){A : P}(ha: A LiesInt seg): A LiesInt ray  := sorry
 
 /- Given two rays $ray_1$ and $ray_2$ with same direction, if the source of $ray_1$ lies on $ray_2$, and if $A$ is a point on $ray_1$, then $A$ lies on $ray_2$. -/
 theorem
-every_pt_lies_on_ray_of_source_lies_on_ray_and_same_dir : sorry := sorry
+every_pt_lies_on_ray_of_source_lies_on_ray_and_same_dir {ray₁ ray₂ : Ray P}(eqDir: ray₁.toDir=ray₂.toDir)(h: ray₁.source LiesOn ray₂){A : P}(ha: A LiesOn ray₁): A LiesOn ray₂:= sorry
```

Comment:
The name `eqDir` can be renamed more simple as `h` of `e` . We do not require the name of a proposition to be informative. And in general, it should only consist lower-case letters.